pkgs/by-name/so/sonar-scanner-cli/package.nix

```
@@ -0,0 +1,81 @@
+{
+  lib,
+  coreutils,
+  fetchFromGitHub,
+  jre,
+  libarchive,
+  makeWrapper,
+  maven,
+  nix-update-script,
+}:
+
+maven.buildMavenPackage rec {
+  pname = "sonar-scanner-cli";
+  version = "6.2.0.4578";
+
+  src = fetchFromGitHub {
+    owner = "SonarSource";
+    repo = "sonar-scanner-cli";
+    rev = "refs/tags/${version}";
+    hash = "sha256-T7Vx87TjLMUWLYkKKu1pZfXvR3LI+rmTfVeuvEz0qeM=";
+  };
+
+  mvnHash = "sha256-IKwpTUEE+WHl4WLkxyGvBmHo5tF9dvhkFobIZ8VAtv4=";
+
+  mvnParameters = "-Dproject.build.outputTimestamp=1980-01-01T00:00:02Z";
+
+  nativeBuildInputs = [
+    coreutils
+    libarchive
```

Do we link against libarchive or use it to unpack things? If the first we need to move it to buildInputs.

Or is it used for the bsdtar below? I am not sure if there is a tar in stdenv we could use already.

---

Good to know. It is totally fine to use it here but a short little comment could clear this up.

```suggestion
    libarchive # for bsdtar
```